### DISCUSSION TITLE
[Permission] Create a Mobile Version of bridge.

### DISCUSSION THREAD

**Discussion by Gekocaretaker** [bridge-core/bridge.#258] [Ideas]
Dear solvedDev,
    I was wondering if I could make a mobile version of bridge.
My plan is to call it gateway. as a similarity to bridge. but I would gladly change it if you want it different.
I am planning on hosting it on github for easy access, but I will change it if you say so.

I am hoping that you are alright with a mobile version as this has been a requested feature for quite some time.

Sincerely,
    gekocaretaker

**Comment by solvedDev** [comment]
Please just contact me on Discord/Twitter :)
